src/components/productCard/ProductCard.component.tsx

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from 'react';
+
+import cart from '@/assets/icons/cart.svg';
+import type Product from '@/interface/product.ts';
+
+import styles from './productCard.module.css';
+
+const ProductCard = ({
+    product,
+    increaseCounter,
+    decrementCounter,
+}: {
+    product: Product;
+    increaseCounter: () => void;
+    decrementCounter: () => void;
+}) => {
+    const [isCart, setIsCart] = useState(false);
+
+    useEffect(() => {
+        const storedValue = localStorage.getItem(product.id.toString());
+        if (storedValue !== null) {
+            const isProductInCart = JSON.parse(storedValue);
+            setIsCart(isProductInCart);
+        }
+    }, [product.id]);
+    const changeCart = () => {
+        setIsCart(!isCart);
+        localStorage.setItem(product.id.toString(), JSON.stringify(!isCart));
+        if (!isCart) {
```

Comment:
use ternary operator. It more readable and short 

`isCart ? decrementCounter() : increaseCounter()`

Reply:
fixed